client/src/App.jsx

```diff
@@ -34,7 +36,12 @@ export default class App extends Component {
 
   componentDidMount() {
     this.setState(ticketsSample);
-  }
+    socket.emit("getmails");
+    socket.on("mails", data => {
+      this.setState({
+        receivedMails: this.state.receivedMails + JSON.parse(data).html
```

Comment:
There exists a state that is used across the application (`tickets`), why create a new one?
```suggestion
        tickets: this.state.receivedMails + JSON.parse(data).html
```